examples/worlds/trajectory_follower.sdf

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition Gazebo trajectory follower plugin demo.
```

Comment:
This example doesn't seem to be working. On Fortress the boxes start moving immediately, but here they don't.

Reply:
I have a fix. Will push soon with other fixes.

Reply:
I don't know why, but on Garden we needed to explicitly enable the `WorldPose` coponent, see 7313edea5a1ab75933cf23e40af8eef7ae94ea45.

There may be something else enabling that component on Fortress. CC @caguero